### PR TITLE
SendGrid and Mailjet example code

### PR DESCRIPTION
**Pull request by joefearnley**
Hi @peter279k ,

I have added SendGrid and Mailjet example code in the README. Let me know if I missed anything.

Thanks,
Joe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,41 @@ It's based on [Guzzle](https://github.com/guzzle/guzzle),HTTP client.
...
+  $config = [
+      'api-key' => 'sendgrid-api-key',
+      'from-name' => 'John Doe',
+      'from-email' => 'john.doe@gmail.com',
+      'to-name' => 'John Doe',
+      'to-email' => 'john.doe@gmail.com',
+      'subject' => 'SendGrid Test',
+      'contents' => 'Sendgrid is awesome!',
+  ];
+  $service = (new ServiceFactory)->create('SendGrid');
```
Comment (peter279k) [inline README.md]: The ```ServiceFactory``` missed the namespace and it should be added in the head of sample code.

File: README.md [hunk 2]
```diff
@@ -54,6 +54,41 @@ It's based on [Guzzle](https://github.com/guzzle/guzzle),HTTP client.
...
+      'api-key-public' => 'mailjet-public-api-key',
+      'api-key-private' => 'mailjet-privave-api-key',
+      'from-name' => 'John Doe',
+      'from-email' => 'john.doe@gmail.com',
+      'to-name' => 'John Doe',
+      'to-email' => 'john.doe@gmail.com',
+      'subject' => 'Mailjet Test',
+      'contents' => 'Mailjet is awesome!',
+  ];
+  $service = (new ServiceFactory)->create('Mailjet');
```
Comment (peter279k) [inline README.md]: The namespace is the same problem.

**Review by peter279k — CHANGES_REQUESTED** [review]

**Comment by peter279k** [comment]
Hi @joefearnley, thank you for your PR.
I'm sorry that I cannot review this right away.
And I will have twelve days of the military training from the beginning of the next week.
Please feel free to send PR and I will review them after finishing the training.

Thanks.

**Comment by joefearnley** [comment]
Hi @peter279k ,

No problem, review them this when you get time. I updated this adding the namespaces to the README examples for SendGrid and Mailjet.

**Review by peter279k — APPROVED** [review]

**Comment by peter279k** [comment]
I'm sorry that the review is delay.
Thank you for your PR!

**Pull request merged**